x-pack/plugins/cases/public/methods/translations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const CASE_SUCCESS_TOAST = (title: string) =>
```

Comment:
Let's move them in the same folder with the `useCasesToast` hook when you move it in the `components` folder.